core/kotlinx-coroutines-core/src/main/kotlin/kotlinx/coroutines/experimental/channels/BroadcastChannel.kt

```diff
@@ -47,7 +47,7 @@ public interface BroadcastChannel<E> : SendChannel<E> {
      * The resulting channel shall be [closed][SubscriptionReceiveChannel.close] to unsubscribe from this
      * broadcast channel.
      */
-    public fun openSubscription(): SubscriptionReceiveChannel<E>
+    override fun openSubscription(): SubscriptionReceiveChannel<E>
```

Comment:
You can remove this method and shift documentation into `Subscribable`'s one.

Reply:
Yes I'll do it.

Reply:
Mmm. After a 2nd though I wonder if removing it couldn't cause binary incompatibilities?

Should I leave it for this reason?

Reply:
Yes it could definitely cause binary incompatibility. I think we should leave it.